docs/UserGuide.md

```diff
@@ -465,11 +472,54 @@ Select a details panel to display. Available panels:
+
+The same idea goes for deciding the goal deadline for monthly and yearly. A monthly goal's deadline will always fall on the last day of the month. 
+A yearly goal's deadline will always fall on the last day of the year. To give another example, assume that you have a set a monthly goal of meeting your friend. 
+The last time you met your friend was on the 13th March. Since you have already achieved your goal of meeting your friend for the month of March, you have to meet your friend
+again by the 30th April to ensure that you always meet your friend once a month.
+
+## Streaks
+
+You should have a good understanding of how goal deadlines are calculated before reading this section. The section for it is [here](#how-are-goal-deadlines-calculated).
+
+### The dashboard
+
+![StreaksDashboard](images/StreaksDashboard.png)
+
+You will be rewarded if you managed to stick to your goal consistently. You can maintain a streak (think of it as Snapchat's Streak feature) with a friend if you have been meeting him/her regularly.
+To ensure that you are able to see your friend's name on the Streaks dashboard, they need to have a goal set to them first. More information on setting goal can be found [here](#setting-meeting-goal-set-goal). 
+To reiterate, your friend will only appear on the dashboard if you explicitly set a goal. Just recording meetings with them will not work.
+
+The dashboard can be viewed by running the `view streaks` command. The Streaks dashboard will show up on the details panel on the right of FriendDex. 
+It will contain a list of friends and their streak indicated by a number beside their name. The dashboard will be sorted in descending order with the friend that has the highest streak right at the top.
+
+### How are streaks calculated?
+
+For those who have not used Snapchat before, do not fear. We will try our best to give a detailed explanation here. 
+
+Streaks are calculated relative to the date today. A year has approximately 52 weeks. Assume that your friend, John, has a weekly meeting goal set and the current week is week 10, 
+you started to meet your friend on week 7 and you consistently met him for weeks 8 and 9 but not yet on week 10. John's streak will be 3 on week 10. 
+If you managed to meet him on week 10, his streak will be incremented by 1, to 4. However, once the Monday of week 11 reaches, John's streak will 
+be reset back to 0. The goal deadline for weekly goals is 11:59pm on Sundays, and you did not manage to meet John for the whole of week 10. 
+Because of that, you failed to keep to your goal of meeting John weekly and so the streak resets. 
+
+Streaks for monthly and yearly goals are calculated the same way. You can switch the week numbers to months or years. Another thing to note is if you have already achieved the goal
+for that time period (e.g. week) for a particular friend, any additional meetings with that friend during the same period will be added to his/her streak.   
```

Comment:
Is this "will be added" or "will not be added"?

Reply:
Yeap, it will be added. Adding meetings will always increase the streak if the meeting is qualified to be counted, i.e. in the same period of the first meeting that starts the streak up till today.